frontend/src/store/items.ts

```diff
@@ -30,46 +30,42 @@ class ItemsStore {
    * == GETTERS ==
    */
   public getItemById = (id: string | undefined): BaseItemDto | undefined => {
-    return computed(() => (id ? state.value.byId[id] : undefined)).value;
+    return id ? state.byId[id] : undefined;
   };
```

Comment:
Do you know if I can get the shorter version syntax I like without the `return`?